packages/lesswrong/lib/collections/posts/helpers.ts

```diff
@@ -6,6 +6,7 @@ import { userGetDisplayName } from '../users/helpers';
 import { postStatuses, postStatusLabels } from './constants';
 import { cloudinaryCloudNameSetting } from '../../publicSettings';
 import Localgroups from '../localgroups/collection';
+import moment from '../../moment-timezone';
```

Comment:
As you do more things with moment, it might be sooner rather than later that's the best time to switch away from moment, which is ~deprecated: https://momentjs.com/docs/#/-project-status/

**Definitely** not for this PR.

Reply:
Yeah I agree we should do this, not sure how complicated it would be though